### DISCUSSION TITLE
goravel/valet!

### DISCUSSION THREAD

**Discussion by inmanturbo** [goravel/goravel#483] [Ideas]
Honestly the reason I always use php and laravel for rapid prototype and MVP (minimum viable product) is because of laravel/valet and the ease of web development environment and nginx integration.

I actually usually use linux and valet/linux: https://github.com/cpriego/valet-linux

No other framework, not even django, next etc. has anything so nice. 

lol every time I think for instance, "`django` is nice, I could use some nice python libraries for data processing in this new prototype", I decide that no, writing the libraries in php is easier, because I can easily get up and simulate a virtual web host environment with sites interacting over api's etc. Or even just nice to have the `http://projectname.test` up and running so easily and not have to start a server, remember the port, leave open a terminal, etc. just `laravel new`, `valet link` and `valet open`

Same with go. Always rather rewrite with go later (to save resources) if app or product ends up viable and needs to be deployed to serverless or something.

Any interest doing something like valet for goravel?

**Comment by hwbrzzl** [comment]
Yes, this is a great idea, and is a big project. We have no plan for now, but welcome PR. 👍
